tools/travis/before_install.sh

```diff
@@ -97,10 +99,11 @@ echo "<settings>
 echo -e "\nInstalling some python packages: "
 
 pip install --user --upgrade pip
+pip uninstall pylint
 
 for ((i=0;i<$RETRY_COUNT;i++))
 do
-  pip install --user --upgrade lxml paramiko nose texttable ipmisim pyopenssl mock flask netaddr pylint pep8 > /tmp/piplog
+  pip install --user --upgrade lxml paramiko texttable ipmisim pyopenssl mock flask netaddr nose pylint pycodestyle six astroid Markdown > /tmp/piplog
```

Comment:
@rafaelweingartner are we using new pkgs - `six astroid Markdown`, where/how?

Reply:
`Markdown` I added because I was getting a message saying that it is not installed, but some of these dependencies might require it. That is why I installed.

`six and astroid` seems to be used by `pylint` and `nose`. And I think that is the problem we were having. There is an issue regarding these dependencies versions, which are not automatically upgraded when you install some library the relies on them.

`¯\_(ツ)_/¯`

Reply:
@rafaelweingartner ideally, pip install should install additional dependencies we should not need to install dependencies for pylint manually.

Reply:
Another note, the new release of pip (v10) might deprecate few things -- I know python 2.6.x is not well supported now.

Reply:
I think the problems happens when the dependencies are already installed. Then, it seems that for some cases `pip` does not upgrade them if you do not ask to. 

Again, I am not a Python guy, so that that hypothesis with a grain of salt.

Reply:
That is what I expected to happen, but it seems it did not work.